insights/specs/__init__.py

```diff
@@ -156,6 +156,7 @@ class Specs(SpecSet):
     du_dirs = RegistryPoint(multi_output=True, filterable=True)
     dumpe2fs_h = RegistryPoint(multi_output=True)
     duplicate_machine_id = RegistryPoint(filterable=True)
+    eap_json_reports = RegistryPoint(multi_output=True, filterable=True)
```

Comment:
[Important] Please double-check if this spec `filterable=True` or not, since it's a JSON file, to make it `filterable=True`, it's necessary to collect it via `datasource`

Reply:
updated